### ISSUE TITLE
why the readme.md updated ? 

### ISSUE THREAD

**Issue by Tomlk** [fanq15/FSOD-code#8]
because  I find much thing has changed, Please!

**Comment by TachibanaYoshino** [comment]
Have you run through fsod training? Used mask rcnn model?

**Comment by fanq15** [comment]
The old readme has lots of irrelavant contents and now I remove these contents. Please follow the new readme to implement your experiment.

**Comment by Tomlk** [comment]
> The old readme has lots of irrelavant contents and now I remove these contents. Please follow the new readme to implement your experiment.

the pytorch is 0.x.x so , I guess cuda10 can't be used for this project?

**Comment by fanq15** [comment]
Yes, you should use cuda 9.0 and pytorch==0.4.1. Readme is updated.
